### DISCUSSION TITLE
[OP Stack Deployment] How to obtain funds to operate a Sepolia-based OP Stack rollup

### DISCUSSION THREAD

**Discussion by fgimenez** [ethereum-optimism/developers#15] [Deploying the OP Stack] [answered]
### Issue Description

Hello everyone!

My team has deployed an OP-stack rollup rooted on Sepolia, all has been super smooth thanks to the excellent documentation :heart: 

We have obtained a few Sepolia ETH using the community faucets and have topped the admin, sequencer, proposer, and batcher accounts with the amounts suggested in the docs, but now we see that some of them, especially batcher and proposer, are burning a lot of ETH (more when Sepolia is busy of course).

So, what do you think is the long-term solution for getting L1 ETH for operating this rollup? We could try to get as much as possible from the faucets, but I assume that this is not the intended purpose of those funds. I've also checked this issue https://github.com/eth-clients/sepolia/issues/12 not sure if it would be a good idea to try to run sepolia validators or directly request funds on that issue as suggested in some of the comments.

Any help is very welcome, thanks!

**Comment by sbvegan** [comment, marked as answer]
Hey @fgimenez, congrats on getting your OP Stack up and running!

I think that running a Sepolia validator is a good long term solution. 

Soon we're going to be update our [Superchain faucet](https://app.optimism.io/faucet) with Sepolia support. With an Optimist NFT, you'd be able to collect 1 testnet ETH a day, and we'd love to see it used on operating an OP Stack chain. I think there will also be a way to request bulk testnet ether on that page. 

In the meantime, can you share your testnet address? I can send you some from my testing wallet to ease your burden.

**Reply by fgimenez** [reply]
> Hey @fgimenez, congrats on getting your OP Stack up and running!

thanks! :rocket: 

> I think that running a Sepolia validator is a good long term solution.

Great, I'll take a look at the requirements.

> Soon we're going to be update our [Superchain faucet](https://app.optimism.io/faucet) with Sepolia support. With an Optimist NFT, you'd be able to collect 1 testnet ETH a day, and we'd love to see it used on operating an OP Stack chain. I think there will also be a way to request bulk testnet ether on that page.

Awesome, how can I get an Optimist NFT?

> In the meantime, can you share your testnet address? I can send you some from my testing wallet to ease your burden.

Thank you so much! `0x80d5c2E66dafd99C069FEEa0C9300EB421d89308` should be fine, I can distribute from there

**Reply by sbvegan** [reply]
Happy to help, the testnet ether just hit your sepolia account. That should hold you over for a little while! Please reach out if you need additional support running your OP Stack chain or anything else :)
